eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/PlutusScripts.hs

```diff
@@ -5,18 +5,17 @@ import Cardano.Ledger.Alonzo.Language (BinaryPlutus (..), Language (..), Plutus
 import Cardano.Ledger.Alonzo.Scripts (AlonzoScript (PlutusScript), CostModel, mkCostModel)
 import Data.ByteString.Short (pack)
 import Data.Either (fromRight)
-import qualified Data.Map as Map
-import PlutusLedgerApi.Test.EvaluationContext (costModelParamsForTesting)
+import PlutusLedgerApi.Test.V3.EvaluationContext (costModelParamsForTesting)
 
 testingCostModelV1 :: CostModel
 testingCostModelV1 =
   fromRight (error "testingCostModelV1 is not well-formed") $
-    mkCostModel PlutusV1 (0 <$ Map.elems costModelParamsForTesting)
+    mkCostModel PlutusV1 (0 <$ costModelParamsForTesting)
```

Comment:
There are now plutus version specific testing functions, which shoul dbe used instead of all using PlutusV3 version
```suggestion
    mkCostModel PlutusV1 (0 <$ PV1.costModelParamsForTesting)
```